packages/blocks/README.md

```diff
@@ -0,0 +1,65 @@
+# `@viamrobotics/prime-core`
```

Comment:
s/core/blocks/ here and below a few times

Reply:
Fixed!